cmd/keeper/keeper.go

```diff
@@ -1579,6 +1587,7 @@ func main() {
 
 func keeper(cmd *cobra.Command, args []string) {
 	var err error
+	validAuthMethods := map[string]bool{"md5": true, "password": true, "trust": true}
```

Comment:
not a big deal but when a map doesn't require a values I prefer to use `struct{}`: `map[string]struct{}`